Doc/library/multiprocessing.rst

```diff
@@ -1021,6 +1022,8 @@ Miscellaneous
 Connection Objects
 ~~~~~~~~~~~~~~~~~~
 
+.. currentmodule:: multiprocessing.connection
```

Comment:
In the line below

> ``` Therefore, unless the connection object was produced using :func:`Pipe` you ```

``` :func:`Pipe` ``` is not converted to a link because the current module is `multiprocessing.connection`.